lib/src/server/handler.dart

```diff
@@ -410,7 +423,11 @@ class ServerHandlerImpl extends ServiceCall {
       ..onDone(_onDone);
   }
 
-  void _sendError(GrpcError error) {
+  void _sendError(GrpcError error, [StackTrace? trace]) {
+    if (_errorHandler != null) {
+      _errorHandler!(error, trace);
```

Comment:
This could simply be: `_errorHandler?.call(error, trace)` - then `if` is not needed.

Reply:
nice...I never think to use `call()`